### DISCUSSION TITLE
Recursive conversions of sigma rules in to subdirectories

### DISCUSSION THREAD

**Discussion by Iqi-Malick** [SigmaHQ/sigma#3239] [Q&A] [answered]
I want to do implement recursive conversions of sigma rules. Currently conversion for subdirectories is not supported yet. Kindly tell how can I implement recursive conversion? 
@frack113

**Comment by frack113** [comment, marked as answer]
Hi,
For sigmac --recurse AND --ignore-backend-errors ("-rI")
Need the ignore otherwise convertion stop.

On windows this test works : `python sigmac -t es-qs -c .\config\generic\sysmon.yml -rI ..\rules\windows`
